index.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="UTF-8"/>
   <title>User Profile</title>
-  <link rel="stylesheet" type="text/css" href="style.css" />
+  <link rel="stylesheet" type="text/css" href="'dist/style.css'" />
```

Comment:
Get rid of extra quotes here.

But also, is there a reason you are importhing this here from your build folder? This isn't the index.html that will be bundled and served.